docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -21,14 +24,25 @@ Standard Caffe\* layers:
 | InnerProduct | No |
 | Input | No |
 | LRN | No |
+| Normalize | No |
+| Python | No |
```

Comment:
It is actually supported for the particular custom Python operations. I believe that it works for "Proposal" only. But need to double check.

Reply:
Yes, Python extractor works only for operations that have extractor derived from "CaffePythonFrontExtractorOp". Such extractors are implemented for 'rpn.proposal_layer.ProposalLayer' and 'SSH.layers.proposal_layer.ProposalLayer' layers. Should I remove 'Python' from list or write limitation that it works only for Proposal op?

Reply:
Write that it works for Python Proposal op only

Reply:
Done.